contracts/PetPark.sol

```diff
@@ -4,4 +4,59 @@ pragma solidity ^0.8.0;
 
 contract PetPark {
 
-}
\ No newline at end of file
+    address owner;
+
+    enum Gender {Male, Female}
+    enum AnimalType {None, Fish, Cat, Dog, Rabbit, Parrot}
+
+    struct Person {
+        uint Age;
+        Gender gender;
+        }
+
+    mapping (uint => uint) CountByAnimalType;
+    mapping (address => Person) PersonByAddress;
+    mapping (address => uint) BorrowedByAddress;
```

Comment:
Names of variable should start with a lowercase letter

Reply:
I did not fix this one but noted for future